lib/rspec/matchers/dsl.rb

```diff
@@ -521,11 +521,21 @@ def actual_arg_for(block)
         # rspec-rails so that it can define matchers that wrap
         # Rails' test helper methods, but it's also a useful
         # feature in its own right.
-        def method_missing(method, *args, &block)
-          if @matcher_execution_context.respond_to?(method)
-            @matcher_execution_context.__send__ method, *args, &block
-          else
-            super(method, *args, &block)
+        if RUBY_VERSION.to_f >= 2.7
+          def method_missing(method, *args, **kwargs, &block)
+            if @matcher_execution_context.respond_to?(method)
+              @matcher_execution_context.__send__ method, *args, **kwargs, &block
```

Comment:
1.8 won't be able to parse it.
You might be interested in checking this out https://github.com/rspec/rspec-support/pull/392#discussion_r361765961

Reply:
A wrapped the whole thing with a version check so that I don't change anything in older rubies, is that not ok?

Reply:
Oh, I get it now. Ruby 1.8 can't parse the `**` syntax. I'll read the linked issue :+1:.